actors/cache/impl/zcache.go

```diff
@@ -197,13 +202,18 @@ func (m *ZCache) storeShortRobust(short string, robust string) {
 	// Possible ZCache types can be Local or Combined. Both types set the TTL at instantiation time
 	// The ttl here is pointless
 	ctx := context.Background()
-	_ = m.shortRobustMap.Set(ctx, short, robust, DummyTtl)
+	_ = m.shortRobustMap.Set(ctx, short, robust, NotExpiringTtl)
 }
 
 func (m *ZCache) StoreAddressInfo(info types.AddressInfo) {
 	m.storeRobustShort(info.Robust, info.Short)
-	m.storeShortRobust(info.Short, info.Robust)
 	m.storeActorCode(info.Short, info.ActorCid)
+
+	isEvm := strings.EqualFold(info.ActorType, constants.ActorTypeEVM)
+	isEvmAndAddressIsF4 := isEvm && strings.HasPrefix(info.Robust, constants.AddressTypePrefixF4)
+	if !isEvm || isEvmAndAddressIsF4 {
+		m.storeShortRobust(info.Short, info.Robust)
+	}
```

Comment:
Same here. A comment to explain we are skipping the case were the address is an f2 on an EVM actor would be nice